tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/interop/GeneralInteropTest.java

```diff
@@ -235,13 +235,15 @@ public void testFailoverReceivingFromCore() throws Exception {
             }
          }
 
+         Wait.assertEquals(0, queueControl::getDeliveringCount);
```

Comment:
After the reconnection, the deliveringCount should be equal to the prefetchSize.